src/main/java/com/eternalcode/combat/util/MathUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.eternalcode.combat.util;
+
+import java.util.Collection;
+import java.util.function.ToIntFunction;
+
+public class MathUtil {
+
+    private MathUtil() {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
+    }
+
+    public static double getPercentage(int percentage, int baseNumber) {
+        return (percentage / 100f) * baseNumber;
+    }
```

Comment:
metoda jest trochę źle nazwana, nie liczysz tutaj procentów tylko ilość, i chyba można tutaj użyć jeszcze double 
```suggestion
    public static double getCountFromPercentage(int percentage, int totalCount) {
        return (percentage / 100D) * totalCount;
    }
```